### PR TITLE
[shopsys] stock and store management moved to shopsys/framework package

### PR DESCRIPTION
**Pull request by grossmannmartin**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Stock and Store management is a crucial part of the platform, so it should be in the base package
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| Yes 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://mg-move-stock-store.odin.shopsys.cloud
  - https://cz.mg-move-stock-store.odin.shopsys.cloud

### CODE REVIEW

File: packages/framework/src/Controller/Admin/ClosedDayController.php [hunk 1]
```diff
@@ -2,26 +2,25 @@
...
 
 class ClosedDayController extends AdminBaseController
 {
     /**
-     * @param \App\Model\Store\ClosedDay\ClosedDayFacade $closedDayFacade
-     * @param \App\Model\Store\ClosedDay\Grid\ClosedDayGridFactory $closedDayGridFactory
-     * @param \App\Model\Store\ClosedDay\ClosedDayDataFactory $closedDayDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Store\ClosedDay\ClosedDayFacade $closedDayFacade
+     * @param \Shopsys\FrameworkBundle\Model\Store\ClosedDay\Grid\ClosedDayGridFactory $closedDayGridFactory
+     * @param \Shopsys\FrameworkBundle\Model\Store\ClosedDay\ClosedDayDataFactory $closedDayDataFactory
```
Comment (vitek-rostislav) [inline packages/framework/src/Controller/Admin/ClosedDayController.php:23]: controllers can be overridden, so the properties should be protected

File: packages/framework/src/Model/Store/ClosedDay/ClosedDayData.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Store\Store[]
+     */
+    public $excludedStores = [];
+
+    public $domainId;
+
+    public $date;
+
+    public $name;
```
Comment (vitek-rostislav) [inline packages/framework/src/Model/Store/ClosedDay/ClosedDayData.php]: missing annotations?

File: packages/framework/src/Model/Store/OpeningHours/OpeningHoursData.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public $dayOfWeek;
+
+    public $firstOpeningTime;
+
+    public $firstClosingTime;
+
+    public $secondOpeningTime;
+
+    public $secondClosingTime;
```
Comment (vitek-rostislav) [inline packages/framework/src/Model/Store/OpeningHours/OpeningHoursData.php]: annotations?

File: packages/framework/src/Model/Store/StoreData.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+    public $specialMessage;
+
+    public $locationLatitude;
+
+    public $locationLongitude;
+
+    public $image;
+
+    public $uuid;
```
Comment (vitek-rostislav) [inline packages/framework/src/Model/Store/StoreData.php]: annotations

File: packages/framework/src/Resources/config/services.yaml [hunk 5]
```diff
@@ -1039,3 +1039,5 @@ services:
     Shopsys\FrameworkBundle\Component\Messenger\MessageDispatcherDependency:
         arguments:
             $transportDsn: '%env(MESSENGER_TRANSPORT_DSN)%'
+
+    Shopsys\FrameworkBundle\Component\DateTimeHelper\DateTimeHelper: ~
```
Comment (vitek-rostislav) [inline packages/framework/src/Resources/config/services.yaml:1045]: I would consider adding `Helper` suffix to the autoconfiguration part. Along with `Shopsys\FrameworkBundle\Model\Heureka\HeurekaShopCertificationLocaleHelper`, there are two helpers now in the `services.yaml` file :slightly_smiling_face:

**Review by vitek-rostislav — APPROVED** [review]
Hi, nice job there :ok_hand: I have added just a few notes, you can pass it to tests after you address them, I do not need to see the fixes :wink:

File: packages/framework/src/Model/Store/StoreData.php (hunk 4 above)
Comment (grossmannmartin) [reply packages/framework/src/Model/Store/StoreData.php]: added, thanks :)

File: packages/framework/src/Model/Store/OpeningHours/OpeningHoursData.php (hunk 3 above)
Comment (grossmannmartin) [reply packages/framework/src/Model/Store/OpeningHours/OpeningHoursData.php]: added, thanks :)

File: packages/framework/src/Model/Store/ClosedDay/ClosedDayData.php (hunk 2 above)
Comment (grossmannmartin) [reply packages/framework/src/Model/Store/ClosedDay/ClosedDayData.php]: added, thanks :)

File: packages/framework/src/Controller/Admin/ClosedDayController.php (hunk 1 above)
Comment (grossmannmartin) [reply packages/framework/src/Controller/Admin/ClosedDayController.php:23]: fixed :)

File: packages/framework/src/Resources/config/services.yaml (hunk 5 above)
Comment (grossmannmartin) [reply packages/framework/src/Resources/config/services.yaml:1045]: as we discussed, helper may not be a service

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**